scripts/eachdist.py

```diff
@@ -724,16 +726,15 @@ def format_args(args):
     format_dir = str(find_projectroot())
     if args.path:
         format_dir = os.path.join(format_dir, args.path)
-    root_dir = str(find_projectroot())
     runsubprocess(
         args.dry_run,
-        ("black", "--config", f"{root_dir}/pyproject.toml", "."),
+        ("ruff", "check", "--fix"),
         cwd=format_dir,
         check=True,
     )
     runsubprocess(
         args.dry_run,
-        ("isort", "--settings-path", f"{root_dir}/.isort.cfg", "--profile", "black", "."),
+        ("ruff", "format"),
```

Comment:
Have you checked it is finding the config file at the root of the dir?

Reply:
Yes! 

You can run in any directory with verbose mode `ruff format instrumentation/opentelemetry-instrumentation-wsgi -v` for example

```
[2024-10-22][13:11:17][ruff::resolve][DEBUG] Using configuration file (via parent) at: opentelemetry-python-contrib/pyproject.toml
```
>Similar to [ESLint](https://eslint.org/docs/latest/use/configure/configuration-files#cascading-configuration-objects), Ruff supports hierarchical configuration, such that the "closest" config file in the directory hierarchy is used for every individual file, with all paths in the config file (e.g., exclude globs, src paths) being resolved relative to the directory containing that config file. 

https://docs.astral.sh/ruff/configuration/#config-file-discovery